GSA/plainlanguage.gov
# POAM: April 24

This PR handles security vulnerabilities for April 2024

- Addressing critical variability with both Bable updates-> https://github.com/GSA/plainlanguage.gov/security/dependabot/49
- [Chang log](https://github.com/webpack/webpack/releases) for Webpack update.

Testing:
- Pull down branch and run `npm install` and ensure no build errors 
- Start local with `npm run start` and visit local and visually inspect site to ensure no regressions
- Local run `npm run build-js` to ensure no accessibility errors

DependenciesDependency | Old | New
-- | -- | --
 @babel/core | 7.23.7  | 7.24.4 
@babel/preset-env | 7.23.8 | 7.24.4
webpack | 5.89.0 | 5.91.0
